pkg/cmd/adm/unregister_member_test.go

```diff
@@ -65,9 +65,9 @@ func TestUnregisterMemberCallsRestart(t *testing.T) {
 	ctxAct := clicontext.NewCommandContext(term, newClient)
 	called := 0
 	// when
-	err := UnregisterMemberCluster(ctxAct, "member1", func(ctx *clicontext.CommandContext, restartClusterName string) error {
+	err := UnregisterMemberCluster(ctxAct, "member1", func(ctx *clicontext.CommandContext, restartClusterName string, cfcGetter ConfigFlagsAndClientGetterFunc) error {
 		called++
-		return mockRestart(ctx, restartClusterName)
+		return mockRestart(ctx, restartClusterName, getConfigFlagsAndClient)
```

Comment:
it is not actually called so we can just pass nil right ? 
```suggestion
		return mockRestart(ctx, restartClusterName, nil)
```

Reply:
no i want to check the `getConfigFlagsAndClient` func  in mock restart, that it just give error